cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/TestCommand.java

```diff
@@ -133,8 +133,8 @@ public void execute() {
                 Path findRoot = ProjectDirs.findProjectRoot(sourceRootPath);
                 if (null == findRoot) {
                     CommandUtil.printError(errStream,
-                            "Please provide a Ballerina file as a " +
-                                    "input or run build command inside a project",
+                            "Provides a Ballerina file as a " +
```

Comment:
This error message seems wrong. Can you check with the relevant person what this message is supposed to convey?

Reply:
Corrected as follows:

```
"provide a Ballerina file as the " +
                                    "input or the run build command from inside a Ballerina project.
```